### ISSUE TITLE
Virtua Striker 2002 (Triforce) game name

### ISSUE THREAD

**Issue by SieKensou** [FIX94/Nintendont#492]
Hi again, sorry to bother you again about this. I think FIX94 didn't read my last message about the Triforce game names, it was about Virtua Striker 2002 to be exact... In that message of the closed thread I was triying to show you more proofs to make you see that the right name of the game is just "Virtua Striker 2002" and not "Virtua Striker 3 ver. 2002" as you put it in the Nintendont name list.

1.
https://www.youtube.com/watch?v=Pzq4oNKRfZI
Minute 2:04... the title screen says: "Virtua Striker 2002"

2.
Picture of the Triforce disc:
![27948278-960c1c06-62f9-11e7-9c19-94e7e4778a96](https://user-images.githubusercontent.com/28622717/31555873-ffabfb86-b042-11e7-9b7e-e8c601cc5809.jpg)
It says: "Virtua Striker 2002"

3.
Promotional poster of the Triforce arcade game:
![vs2002](https://user-images.githubusercontent.com/28622717/31556360-feaf7e86-b044-11e7-8f1e-d2ad10ca18ae.jpg)
The title says "Virtua Striker 2002".

4.
And wait... let's see your youtube video of the "Triforce Virtua Striker 3 Ver. 2002 Intro"... go to minute 1:08... What can you read at the title screen?
https://youtu.be/6L-5uF7UyEw?t=1m8s
Again... just "Virtua Striker 2002"....

5.
https://en.wikipedia.org/wiki/Virtua_Striker
In the section "history", you can read:
Virtua Striker 3 (2001)
    -Virtua Striker 3 ver. 2002 (Nintendo GameCube port; 2002)
    -Virtua Striker 2002 (2002)

So, basically, Virtua Striker 3 ver. 2002 is the GameCube game, and Virtua Striker 2002 is the Trifroce arcade game... So it is not exactly the same game and they have different names.

As you named it now, both, the GameCube game and the Triforce game appears as Virtua Striker 3 ver. 2002, so you can't know wich one is one or another. Calling the arcade Triforce game with its real name "Virtua Striker 2002" will make you perfectly difference between one and another, so that is a plus of having the right name of the game.
